test/payment/SplitPayment.test.js

```diff
@@ -37,58 +37,58 @@ contract('SplitPayment', function ([_, owner, payee1, payee2, payee3, nonpayee1,
       await ethSendTransaction({ from: owner, to: this.contract.address, value: amount });
 
       const balance = await ethGetBalance(this.contract.address);
-      balance.should.be.bignumber.equal(amount);
+      balance.should.be.bignumber.eq(amount);
     });
 
     it('should store shares if address is payee', async function () {
       const shares = await this.contract.shares.call(payee1);
-      shares.should.be.bignumber.not.equal(0);
+      shares.should.be.bignumber.not.eq(0);
     });
 
     it('should not store shares if address is not payee', async function () {
       const shares = await this.contract.shares.call(nonpayee1);
-      shares.should.be.bignumber.equal(0);
+      shares.should.be.bignumber.eq(0);
     });
 
     it('should throw if no funds to claim', async function () {
-      await expectThrow(this.contract.claim({ from: payee1 }), EVMThrow);
+      await expectThrow(this.contract.claim({ from: payee1 }), EVMRevert);
     });
 
     it('should throw if non-payee want to claim', async function () {
       await ethSendTransaction({ from: payer1, to: this.contract.address, value: amount });
-      await expectThrow(this.contract.claim({ from: nonpayee1 }), EVMThrow);
+      await expectThrow(this.contract.claim({ from: nonpayee1 }), EVMRevert);
     });
 
     it('should distribute funds to payees', async function () {
       await ethSendTransaction({ from: payer1, to: this.contract.address, value: amount });
 
       // receive funds
       const initBalance = await ethGetBalance(this.contract.address);
-      initBalance.should.be.bignumber.equal(amount);
+      initBalance.should.be.bignumber.eq(amount);
 
       // distribute to payees
       const initAmount1 = await ethGetBalance(payee1);
       await this.contract.claim({ from: payee1 });
-      const profit1 = await ethGetBalance(payee1) - initAmount1;
-      assert(Math.abs(profit1 - web3.toWei(0.20, 'ether')) < 1e16);
+      const profit1 = (await ethGetBalance(payee1)).sub(initAmount1);
+      profit1.sub(web3.toWei(0.20, 'ether')).abs().should.be.bignumber.lt(1e16);
```

Comment:
This is such a weird test.

Reply:
Yup, we need to revise all tests at some point, some have poor coverage/are a bit wonky.